app/controllers/concerns/curation_concerns/curation_concern_controller.rb

```diff
@@ -59,6 +56,7 @@ def show_presenter
   end
 
   def edit
+    presenter
```

Comment:
We don't want to use a Solr document as the backer for the presenter on an edit. It should use an ActiveFedora object instead.  The ActiveFedora object may have more data in it than the Solr object, which needs to go on the form.